resources/views/upload.blade.php

```diff
@@ -629,28 +656,36 @@ class="relative flex items-center leading-5 list-inside even:bg-gray-50 rounded
 
 				{{-- STEP 3 --}}
 				<template x-if="step == 3">
-					<div class="" x-show="step == 3">
+					<div x-show="step == 3">
 						<h2 class="font-title text-2xl mb-5 text-primary font-medium uppercase">
 							@lang('app.download-links')
 						</h2>
 
 						{{-- Preview link --}}
-						<div class="flex flex-wrap items-center">
+						<div class="flex flex-wrap items-center relative">
 							<div class="w-1/3 text-right px-2">
 								@lang('app.preview-link')
 							</div>
-							<div class="w-2/3 shadow">
-								<input x-model="bundle.preview_link" class="w-full bg-transparent text-slate-700 h-8 px-2 py-1 rounded-none border border-primary-superlight outline-none" type="text" readonly x-on:click="selectCopy($el)" />
+							<x-tooltip x-show="copynotify.preview" x-on:click.away="copynotify.preview = false" />
+							<div class="w-2/3 shadow flex items-center">
+								<input id="copy-preview" x-model="bundle.preview_link" class="w-full bg-transparent text-slate-700 h-8 px-2 py-1 rounded-none border border-primary-superlight outline-none" type="text" readonly x-on:click="selectCopy($el)" />
```

Comment:
- Missing translation for "Open in new tab"
- Move the input field border to include the icon: `border` is moved to the parent div

My proposal: 
```
<div class="w-2/3 shadow flex items-center border border-primary-superlight">
   <input id="copy-preview" x-model="bundle.preview_link" class="w-full bg-transparent text-slate-700 h-8 px-2 py-1 rounded-none outline-none" type="text" readonly x-on:click="selectCopy($el)" />
   <a class="p-1" title="@lang('app.open-in-new-tab')" :href="bundle.preview_link" target="_blank">
        <x-newtab />
    </a>
</div>
```